Chess/src/King.cpp

```diff
@@ -0,0 +1,101 @@
+
+#include "King.h"
+#include <algorithm>
+#include <functional>
```

Comment:
Avoid unnecessary includes: The <algorithm> and <functional> headers are not needed in the given code. You can remove them since they are not used.